examples/onepass_mean_var_std.py

```diff
@@ -0,0 +1,84 @@
+"""
+Example: One Pass Mean Var Std
+==============================
+
+This example shows how to calculate mean, variance, and standard deviation of log normalized
+gene expression count data in one pass [1].
+
+Example run::
+    python examples/onepass_mean_var_std.py --accelerator gpu --num_workers 4 \
+            --default_root_dir runs/onepass
+
+**References:**
+
+    1. *Algorithms for calculating variance*,
+       (https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance)
+"""
+
+import argparse
+
+import pytorch_lightning as pl
+import torch
+
+from scvid.data import (
+    DistributedAnnDataCollection,
+    IterableDistributedAnnDataCollectionDataset,
+    collate_fn,
+)
+from scvid.module.onepass_mean_var_std import OnePassMeanVarStd
+from scvid.train.training_plan import DummyTrainingPlan
+from scvid.transforms import ZScoreLog1pNormalize
+
+
+def main(args):
```

Comment:
You are welcome: https://click.palletsprojects.com/en/8.1.x/

Reply:
Click is an awesome library! I've used Typer before which is based on Click. Would be nice to incorporate one of these in the future.